src/components/MessageLogger/MessageList.tsx

```diff
@@ -9,7 +9,15 @@ interface Props {
 
 export const MessageList = ({ messages }: Props) => {
   return (
-    <div id={styles.containerMessages}>
+    <div
+      id={
+        messages.length <= 0
```

Comment:
The only thing that changes with this styles is the color, so pass it as a prop like i commented on the LineMessage.tsx